ramls/schema/select.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "select values list schema",
+  "type": "object",
+  "description": "select  metadata",
+  "properties": {
+    "headingTypes": {
```

Comment:
Should this be `headerTypes` instead of `headingTypes`? I think we should be consistent and the API is `/marccat/auth-header-types`. Also, should this field be `authHead[er|ing]Types`?

Reply:
In my opinion it should be headerTypes. 
The classes should be called: HeaderTypeCollection and HeaderType because it creates confusion with the headings
 If you make these changes you have to think about the UI.

Reply:
It is true. I think that it is better have the same name: `headerTypes`. It would also have to be changed in the bibliographic, @carmentrazza. Could we change in other PR?

Reply:
Yes I will do it in another PR because you have to change the UI too.